host/CMakeLists.txt

```diff
@@ -307,7 +307,7 @@ endif()
 add_dependencies(oehost syscall_untrusted_edl)
 add_dependencies(oehost tee_untrusted_edl)
 if(OE_SGX)
-  add_dependencies(oehost sgx_untrusted_edl switchless_untrusted_edl)
+  add_dependencies(oehost sgx_untrusted_edl)
```

Comment:
Accidental removal of switchless_untrusted_edl?

Reply:
Really bad accident, thanks.